package.json

```diff
@@ -28,6 +28,17 @@
     "engines": {
         "node": ">=4.2.0"
     },
+    "files": [
+        "./bin",
+        "./lib",
+        "!./lib/enu",
+        "./loc",
+        "./AUTHORS.md",
+        "./LICENSE.txt",
```

Comment:
This already has it - but do we need `CopyrightNotice.txt`?

Reply:
See my PR description; this file is a JS comment that the build prepends to JS files during build. I don't think it should be in the package.

Reply:
You already mentioned this I suppose. Makes sense.